packages/calcite-components/src/components/chip/chip.tsx

```diff
@@ -340,7 +340,7 @@ export class Chip
           [CSS.selectIconActive]: this.selectionMode === "multiple" || this.selected,
         }}
       >
-        <calcite-icon icon={icon} scale={getIconScale(this.scale)} />
+        {icon ? <calcite-icon icon={icon} scale={getIconScale(this.scale)} /> : null}
```

Comment:
This is only the selection mode icon - the `icon` property icon is already conditionally rendered. How are they experiencing the error? Maybe the `this.selectionMode !== "none"` isn't catching all cases?

I don't think the `icon` property should be required, Chips only have icons specified occasionally.

Reply:
Ah, yeah makes sense, its the "inactive" selection icon. This looks to be a safe change, I can't see any issues with animation or anything in local testing with the proposed change.

The `icon` property is separate.

Reply:
Yeah, not the property. I was thinking maybe we make the `icon` property on `calcite-icon` required to avoid this though?

The issue is that the JSAPI screenshot tests looks for calcite-icon and waits for it to have a path before proceeding. In this case it never gets a path.

cc @jcfranco

Reply:
Oh, sorry, missed the "required on Calcite Icon" part. Carry on :)

Maybe if no icon is provided to Calcite Icon, we render "blank" icon instead of making required? In this case it would have reserved space but maybe it's a better fallback.

Reply:
yeah that could work as well.